### ISSUE TITLE
Add PatientAge, PatientSex, PatientWeight and PatientSize 

### ISSUE THREAD

**Issue by bcli4d** [ImagingDataCommons/IDC-WebApp#1313]
Add these to the DB so they are recognized by collex_metadata_utils/get_bq_metadata() and the API can then expose them as filters and fields.

**Comment by s-paquette** [comment]
@bcli4d A quick note here, Age and Weight are both currently stored as Strings in dicom_all, so that's how they'll be in the pivot. If you want them to be numbers for more logical search operations they need to be converted in dicom_all.

**Comment by bcli4d** [comment]
Yeah, it would be nice if users could search on a range of age or weight. But converting age to a numeric value is problematic. Perhaps our position should be that, if a user really wants to do that, they start from the SQL and add their own conversion.

**Comment by fedorov** [comment]
Per discussion, we will revisit this after v17.
